data/strings/strings.txt

```diff
@@ -30845,3 +30845,233 @@
     vi = Mở trong ứng dụng khác
     zh-Hans = 在另一个应用程序中打开
     zh-Hant = 在另一個應用程式中打開
+
+  [bookmarks_recently_deleted]
```

Comment:
@organicmaps/translations Please review and check if "Lists" is translated in the same way as here for consistency:

```
  [bookmark_lists]
    tags = android,ios
    en = Lists
    af = Lyste
    ar = قوائم
    az = Siyahılar
    be = Спісы
    bg = Списъци
    ca = Llistes
    cs = Seznamy
    da = Lister
    de = Listen
    el = Λίστες
    es = Listas
    es-MX = Listas
    et = Loetelud
    eu = Zerrendak
    fa = لیست‌ها
    fi = Luettelot
    fr = Listes
    he = רשימות
    hi = सूचियों
    hu = Listák
    id = Daftar
    it = Elenchi
    ja = リスト
    ko = 기울기
    lt = Sąrašai
    mr = याद्या
    nb = Lister
    nl = Lijsten
    pl = Listy
    pt = Listas
    pt-BR = Listas
    ro = Liste
    ru = Списки
    sk = Zoznamy
    sv = Listor
    sw = Orodha
    th = รายการ
    tr = Listeler
    uk = Списки
    vi = Danh sách
    zh-Hans = 列表
    zh-Hant = 清單
```

Reply:
`he` is good.

Reply:
It seems like all translations is correct